src/vulkan/image_descriptor.cc

```diff
@@ -101,7 +107,7 @@ void ImageDescriptor::UpdateDescriptorSetIfNeeded(
     layout = VK_IMAGE_LAYOUT_GENERAL;
 
   VkDescriptorImageInfo image_info = {
-      nullptr,  // TODO(asuonpaa): Add sampler here later if used
+      VK_NULL_HANDLE,  // TODO(asuonpaa): Add sampler here later if used
```

Comment:
Does this pass the linter? I'd expect it to complain about it not being a `nullptr`

Reply:
It's passing. Some Android builds break with nullptr.